README.md

```diff
@@ -22,7 +22,9 @@ $ yarn add generate-template-files
 
 ## Usage
 
-Check out the `examples` folder or create a file called `generate.js`. The file can be named anything you want. In that file add in the example code below. Then you can do `node generate.js` once you added your template files.
+1. Check out the `examples` folder or create a file called `generate.js`. Note that this file name is flexible but must be a `.js` file.
```

Comment:
Not sure if it should say `must be a '.js' file`. The user could be using TypeScript.

Maybe it should just say
`Note that this file name is flexible.`

Reply:
Oooh, I agree.